docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -614,69 +428,20 @@ We need coverage rate and efficacy rate to solve for all the variables stratifie
     - Effect size application
     - Value, source
     - Note
-  * - :math:`r2_{ux_{uncom}}`
-    - transition probability
-    - recovery probability of untreated uncomplicated SAM among 6-59 months
-    - 
-    - 
-    - 
-    - 
-  * - :math:`r2_{tx_{uncom}}`
+  * - :math:`r2_{ux}`
     - transition probability
-    - recovery probability of treated uncomplicated SAM among 6-59 months
-    - :math:`E_{r2_{6-59_{otp}}}` recovery efficacy of OTP for uncomplicated SAM among 6-59 months (>1)
-    - :math:`r2_{ux_{uncom}}\times(E_{r2_{6-59_{OTP}}}) = r2_{tx_{uncom}}`
-    -
-    - May get :math:`r2_{tx_{uncom}}` from programmatic data/studies
-  * - :math:`d1_{ux_{uncom}}`
-    - mortality probability
-    - probability of death of untreated uncomplicated SAM among 6-59 months 
-    - 
+    - Daily recovery prbability of untx SAM
+    - :math:`\frac{1}{\text{median time-to-recovery of utx SAM}}`
     - 
     - 
     - 
-  * - :math:`d1_{tx_{uncom}}`
-    - mortality probability
-    - probability of death of treated uncomplicated SAM among 0-6 months
-    - :math:`E_{d1_{6-59_{otp}}}` mortality efficacy of OTP for SAM among 6-59 months (<1)
-    - :math:`d2_{ux_{uncom}}\times(E_{d1_{6-59_{OTP}}}) = d1_{tx_{uncom}}`
-    -
-    - May get :math:`d1_{tx_{uncom}}` from programmatic data/studies
-  * - :math:`r2_{ux_{com}}`
+  * - :math:`t1_{sam}`
     - transition probability
-    - recovery probability of untreated complicated SAM among 6-59 months
-    - 
-    - 
+    - Daily recovery probability of tx SAM 
+    - :math:`\frac{1}{\text{median time-to-recovery of tx SAM}}`
```

Comment:
This doesn't match my understanding of how the intervention is going to work. The probability of moving along the `t1_sam` arrow shouldn't have anything to do with the time-to-recovery of SAM, but only the probability that you eventually recover once you get treatment. Unless you've changed the modeling strategy.

@aflaxman does this look right to you?

Reply:
> This doesn't match my understanding of how the intervention is going to work. The probability of moving along the `t1_sam` arrow shouldn't have anything to do with the time-to-recovery of SAM, but only the probability that you eventually recover once you get treatment. Unless you've changed the modeling strategy.
> 
> @aflaxman does this look right to you?

I think that this is still work-in-progress

Reply:
the probability that you eventually recover from SAM is another way to say time to recovery. How you recover from SAM determines how long you stay in the SAM box. So the probability of eventually recover after getting treated is say coverage x recovery probability = 0.48*.75 = 0.36. This probability of 0.36 means you will on average stay in the SAM box for 2.7 days before recovering... so the time to recovery and probability of recovery is just really a different way to say 'time-to-recovery'.

Reply:
a probability of recovery of 0.36 is way too fast. If people are only staying in the SAM box for 2.7 days, then the incidence will have to be super high to compensate for this short duration.